cmd/flag/stats_flag.go

```diff
@@ -23,7 +23,7 @@ type StatsFlag struct {
 
 // Add flags to the command.
 func (f *StatsFlag) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information. (default false)")
+	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information and generates pass reports. (default false)")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&f.ShowStats, "stats", "", false, "Output telemetry stats information and generate pass summaries. (default false)")
```

Reply:
I feel like `report` sounds like it might contain an evaluation of the pass (quality, etc)